src/Client/WebToElasticms/Filter/Html/TagCleaner.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Client\WebToElasticms\Filter\Html;
+
+use App\Client\WebToElasticms\Config\ConfigManager;
+use App\Client\WebToElasticms\Config\WebResource;
+use Symfony\Component\DomCrawler\Crawler;
+
+class TagCleaner implements HtmlInterface
+{
+    public const TYPE = 'tag-cleaner';
+    private ConfigManager $config;
+
+    public function __construct(ConfigManager $config)
+    {
+        $this->config = $config;
+    }
+
+    public function process(WebResource $resource, Crawler $content): void
+    {
+        foreach ($this->config->getCleanTags() as $cleanTag) {
+            foreach ($content->filter($cleanTag) as $item) {
```

Comment:
The filter supports css selector:
```suggestion
            foreach ($content->filter(\implode(', ', $this->config->getCleanTags())) as $item) {
```